psalm.xml

```diff
@@ -16,10 +15,48 @@
         </ignoreFiles>
     </projectFiles>
     <stubs>
+        <file name="vendor/jetbrains/phpstorm-stubs/PDO/PDO.php" />
         <file name="vendor/jetbrains/phpstorm-stubs/ibm_db2/ibm_db2.php" />
         <file name="vendor/jetbrains/phpstorm-stubs/mysqli/mysqli.php" />
         <file name="vendor/jetbrains/phpstorm-stubs/oci8/oci8.php" />
         <file name="vendor/jetbrains/phpstorm-stubs/pgsql/pgsql.php" />
         <file name="vendor/jetbrains/phpstorm-stubs/sqlsrv/sqlsrv.php" />
     </stubs>
+    <issueHandlers>
+        <InvalidParamDefault>
+            <errorLevel type="suppress">
+                <file name="lib/Doctrine/DBAL/Driver/Connection.php" />
+            </errorLevel>
+        </InvalidParamDefault>
+        <TooFewArguments>
+            <errorLevel type="suppress">
+                <!--
+                    Requires a release of
+                    https://github.com/JetBrains/phpstorm-stubs/commit/300ee558a0fc33491a16a32346886ecd8b6726b5
```

Comment:
Let's better reference a pull request (https://github.com/JetBrains/phpstorm-stubs/pull/727) just in case they decide to rebase the branch.

Reply:
Rebasing master? Who does that (besides Doctrine I mean :trollface: )?

Reply:
I should have used “human-readable” approach. Besides, a PR describes the issue better than just a commit.